eventloop/eventloop.go

```diff
@@ -92,6 +94,16 @@ func WithRegistry(registry *require.Registry) Option {
 	}
 }
 
+// WithJobDoneCallback
+// Called when the next job will not be executed
+// Occurs when all jobs have completed or EventLoop.Stop is called
+// If EventLoop.Start is executed multiple times, the callback will be called repeatedly
+func WithJobDoneCallback(fn func()) Option {
```

Comment:
I'm not sure this will be useful for anything. The Run() and Stop() methods both guarantee that by the time they return no more jobs are running. This does not provide any guarantee because it runs asynchronously, by the time it's called (or during its run) there may be more jobs added.

If you need to know when all jobs are finished you should use the Run() method and wait until it returns.

Reply:
I need to set multiple node instances as a pool. Then run multiple scripts repeatedly. With this callback, I can schedule tasks in multiple instances. When multiple instances are running at the same time, when one of them reports an error or Exit can actively call stop to stop other instances for subsequent tasks, and JobRemainCount is used to control some recovery-related operations,After stopping multiple instances, judge whether there are any follow-up tasks, and if there are tasks, create a new instance
Of course, I can write a js script to complete similar operations, but it will be a lot of trouble

Reply:
I need to submit multiple asynchronous tasks at one time in the node instance. Without this callback, the difficulty of scheduling will be relatively high, because most of the tasks are async
If it is a synchronous task, it is indeed a more convenient choice to pass Run()
Of course, I submit multiple tasks first, and finally use Run(), but there may be difficulties in scheduling between multiple tasks

Reply:
![image](https://user-images.githubusercontent.com/43053461/216880035-27726e83-b172-40e0-a359-ceb46c0113d6.png)
This can indeed achieve the function I want, this pr is really not necessary, thank you